### PR TITLE
Modified READMEs to provide clarity on what the tools do

### PR DESCRIPTION
**Pull request by hemal**

### CODE REVIEW

File: cert/README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ cert
 
 ###### Automatically create and maintain iOS code signing certificates.
 
+Cert is focused exclusively on code signing. You can create a new code signing identity for a certain environment (dev, distribution) and check for any existing valid certificates installed locally.
```
Comment (ohwutup) [inline cert/README.md]: Something sounds a bit weird about this language. Maybe:

"You can create new code signing identities for different environments (development and distribution) and use any existing valid certificates installed locally."

File: deliver/README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ deliver
 
 ###### Upload screenshots, metadata and your app to the App Store using a single command
 
-`deliver` can upload ipa or pkg files, app screenshots and more to iTunes Connect from the command line.
+`deliver` will upload screenshots, metadata and binaries to iTunes Connect. It’s what you use to submit your app for review.
```
Comment (ohwutup) [inline deliver/README.md]: "`deliver` uploads" to match the style of the other README's. Then maybe "Use deliver to submit your app for review."?

File: frameit/README.md [hunk 3]
```diff
@@ -36,6 +36,8 @@ frameit
 
 ###### Quickly put your screenshots into the right device frames
 
+`frameit` allows you to put a gorgeous device frame around your iOS screenshots just by running one simple command. It’s perfect to generate screenshots for the App Store, your website, QA or emails. 
+
```
Comment (ohwutup) [inline frameit/README.md]: Second sentence should be something to the tune of "Use `frameit` to prepare your screenshots for the App Store..." 

`snapshot` generates the screenshots, `frameit` prepares them.

File: match/README.md [hunk 4]
```diff
@@ -38,6 +38,8 @@ match
 
 A new approach to iOS code signing: Share one code signing identity across your development team to simplify your codesigning setup and prevent code signing issues.
 
+`match` is the implementation of the https://codesigning.guide concept. `match` creates all required certificates and profiles and puts them into a separate git repository. Every team member with access to the repo can use those certificates to code sign an app. Automatically repairs broken and expired profiles. This is all about having one repo per dev team.
```
Comment (ohwutup) [inline match/README.md]: The last sentence sounds a bit strange, maybe something like:

"`match` creates all required certificates and provisioning profiles an stores them in a separate git repository. Every team member with access to the repo can use those credentials for code signing. `match` also automatically repairs broken and expired credentials. The easiest way to share signing credentials across teams"

File: pem/README.md [hunk 5]
```diff
@@ -39,6 +39,8 @@ Tired of manually creating and maintaining your push notification profiles for y
 
 `pem` does all that for, just by running `pem`.
 
+`pem` will check if there is a valid push notification profile, and if not, create a new one. When it does this, it will creates a pem, cer and p12 file. Pem does not cover uploading the file to your server
```
Comment (ohwutup) [inline pem/README.md]: "`pem` creates new `.pem`, `.cer`, and `.p12` files to be uploaded to your push server if a valid push notification profile is needed."

File: produce/README.md [hunk 6]
```diff
@@ -35,7 +35,9 @@ produce
 
 ###### Create new iOS apps on iTunes Connect and Dev Portal using your command line
 
-Get in contact with the developer on Twitter: [@FastlaneTools](https://twitter.com/FastlaneTools)
+Produce creates new iOS apps on both the Dev Portal and iTunes Connect. Produce sets the minimum required information for a new app and is designed to be minimalistic in nature.
+
+Get in contact with the developers on Twitter: [@FastlaneTools](https://twitter.com/FastlaneTools)
```
Comment (ohwutup) [inline produce/README.md]: "`produce` creates new iOS apps on both the Apple Developer Portal and iTunes Connect with the bare minimum information."

File: scan/README.md [hunk 7]
```diff
@@ -36,6 +36,8 @@ scan
 
 ###### The easiest way to run tests of your iOS and Mac app
 
+`scan` makes it easy to run tests of your iOS and Mac app on a simulator.
```
Comment (ohwutup) [inline scan/README.md]: This also works on connected devices I believe.

File: screengrab/README.md [hunk 8]
```diff
@@ -35,6 +35,8 @@ screengrab
 
 ###### Automated localized screenshots of your Android app on every device
 
+Generate localised Android screenshots for different device types and languages for Google Play that are ready to be uploaded using ([`supply`](https://github.com/fastlane/fastlane/tree/master/supply).
```
Comment (ohwutup) [inline screengrab/README.md]: `screengrab` generates localized screenshots of your Android app for different device types and languages for Google Play and can be uploaded using [`supply`](https://github.com/fastlane/fastlane/tree/master/supply).

File: snapshot/README.md [hunk 9]
```diff
@@ -35,6 +35,8 @@ snapshot
 
 ###### Automate taking localized screenshots of your iOS app on every device
 
+'snapshot' generates localised iOS screenshots for different device types and languages for the App Store that are ready to be uploaded using ([`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver) is your friend).
```
Comment (ohwutup) [inline snapshot/README.md]: *localized

also, the last part has strange parens. Maybe should be something like "... that can be uploaded using [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver)."

File: supply/README.md [hunk 10]
```diff
@@ -36,6 +36,8 @@ supply
 
 ###### Command line tool for updating Android apps and their metadata on the Google Play Store
 
+`supply` uploads app metadata, screenshots and binaries to Google Play. You can also select tracks for builds and promote builds to production.
```
Comment (ohwutup) [inline supply/README.md]: not sure what tracks are. could just be my lack of android knowledge.

File: cert/README.md (hunk 1 above)
Comment (hemal) [inline cert/README.md]: love this!

File: deliver/README.md (hunk 2 above)
Comment (hemal) [inline deliver/README.md]: That sounds better. Modifying

File: frameit/README.md (hunk 3 above)
Comment (hemal) [inline frameit/README.md]: Modfied.

File: cert/README.md [hunk 11]
```diff
@@ -35,7 +35,7 @@ cert
 
 ###### Automatically create and maintain iOS code signing certificates.
 
-Cert is focused exclusively on code signing. You can create a new code signing identity for a certain environment (dev, distribution) and check for any existing valid certificates installed locally. 
+Cert is focused exclusively on code signing. You can create new code signing identities for different environments (development and distribution) and use any existing valid certificates installed locally.
```
Comment (ohwutup) [inline cert/README.md]: `cert` like all the others?

Comment (hemal) [inline cert/README.md]: fixed

**Comment by ohwutup** [comment]
👍

File: supply/README.md (hunk 10 above)
Comment (KrauseFx) [inline supply/README.md]: A track is either production, beta or alpha:

File: cert/README.md [hunk 12]
```diff
@@ -35,6 +35,8 @@ cert
 
 ###### Automatically create and maintain iOS code signing certificates.
 
+`cert` is focused exclusively on code signing. You can create new code signing identities for different environments (development and distribution) and use any existing valid certificates installed locally.
```
Comment (KrauseFx) [inline cert/README.md]: Why the `exclusively`?

**Comment by KrauseFx** [comment]
Looks good however I noticed that not all tools were updated (e.g. gym).

**Pull request merged**